composer.json

```diff
@@ -11,12 +11,13 @@
     "autoload-dev": {
         "psr-4": {
             "tests\\Libero\\ContentStore\\": "tests/",
-            "tests\\Libero\\JatsContentWorkflow\\": "vendor-extra/JatsContentWorkflow/tests/"
+            "tests\\Libero\\JatsContentWorkflowBundle\\": "vendor-extra/JatsContentWorkflowBundle/tests/"
         }
     },
     "require": {
         "php": "^7.3",
         "ext-ctype": "*",
+        "ext-intl": "*",
```

Comment:
not sure if it's the bundle or the application that should declare the dependency on the `intl` extension? (since the bundle has its own `composer.json`)

Reply:
The bundle depends on https://github.com/libero/content-store/blob/6235d9095e920b373c5e1e5b830d3977bf1329c7/vendor-extra/JatsContentWorkflowBundle/composer.json#L27 which the application replaces with the full extension.